73132 Dashboard introduction (#407)
valtimo-platform/valtimo-documentation: introduction/modules/dashboard-introduction.md

# What are dashboards

Dashboards can provide the user with statistical information.
Using this information, the user will be enabled to prioritize and focus on specific tasks.

A dashboard contains a set of widgets that display data in a customized way. 
These dashboards and widgets can be configured by an administrator accordingly.

## Concepts

* **Dashboard** A container for any number of widgets. 
    Multiple dashboards can be configured, which will be shown to the user as tabs.
* **Widget** Presents data from a datasource using a display-type to the user.

> * **Data-source** Retrieves data from any desired source. 
>     Every datasource can publish output features like `value`, `total` or `values`.

Reviewer: ```suggestion
* **Data-source** Every dashboard widget retrieves its data from a data-source. 
```

I don't think people need to know about response body fields: 'value', 'total', 'values'

Author: You're probably right that the introduction is not the right place for that information.

* **Display-type** A display type is responsible for the visualisation of data. 

>     For example: a number, graph, table, or any custom. 

Reviewer: ```suggestion
    For example: bar char, big number, meter, gauge, or any custom. 
```
